ui/src/app/components/sidenav-view-assessment/sidenav-view-assessment.component.html

```diff
@@ -1,2 +1,101 @@
-<h2>Assessment content coming soon!</h2>
-<button mat-button (click)="closeSidenav()">Close</button>
+<div>
+  <div class="header">
+    <span class="mat-h3 header-title">Assessment report</span>
```

Comment:
https://drive.google.com/file/d/1PAMjagdSRO7ZHSy9ZzrQ0Zb2_MynDdO6/view?usp=sharing&resourcekey=0-idXhZV2UqPtTh_p2fm8u_Q The header is not of same size as in other pages

Reply:
Updated the CSS